aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1 +1,427 @@
+various definitions, a common picture emerges:
+
+| Voice                         | Tone                                        |
+| ----------------------------- | ------------------------------------------- |
+| Who they hear                 | What they hear                              |
+| Personality                   | Attitude                                    |
+| How you are doing the writing | Who they hear                               |
+| Style                         | How your message comes across               |
+| What makes us uniquely us     | How we communicate to our specific audience |
+
+### Voice
+
+Voice is who the reader "hears" delivering messaging; embedded within it is the
+worldview of the author. Voice can be spoken, witty, friendly, formal, etc.
+At HPE, the desire is for HPE's voice to be perceived as intelligent, experienced,
+objective, inclusive, and non-threatening supporting HPE's worldview which
+believes in accelerating what's next, making bold moves, the power of yes we can, and
+being a force for good.
+
+A person's voice typically does not change once established. If HPE's voice is
+inconsistent, people will notice. For example, people notice the difference
+between writing academically when compared to sounding like a 5-year old. This
+kind of inconsistency can result in lack of trust.
+
+### Tone
+
+Tone has to do with what is conveyed or how it comes across. It can express seriousness, urgency, playfulness, objectivity, intimacy, condescension, etc.
+
+Unlike voice, tone can change based on the target audience, context, and subject
+matter. It is situational. For example, tone is very different when giving
+directions during an emergency compared to when ordering a pizza.
+
+Because tone is situational, it may be tweaked to appropriately fit the context
+within an HPE application. While tone may change, the message should be consistently
+delivered in [HPE's voice](#voice) supporting the perception of intelligence,
+experience, objectivity, and inclusivity.
+
+### Point of View
+
+Point of view is the perspective from which the message is written.
+First person perspective uses words like: I, me, us, we, our, etc.
+Second person uses any form of the word "you" which has the effect of
+addressing the reader. Third person directs the reader to things that are
+neither the writer or the reader.
+
+Examples:
+
+- _We noticed the server is malfunctioning._ (1st person-ish)
+- _Your server is malfunctioning._ (2nd person-ish)
+- _The server is malfunctioning._ (3rd person - no focus on the writer or the reader)
+
+## Guidelines for Constructing Good Messaging
+
+The following are some high-level guidelines for constructing good messaging:
+
+- Use HPE's voice and tone
+  - Conversational American English
+  - Correct grammar and punctuation
+  - Active tense
+  - Human and approachable, not robotic sounding
+  - Positive, "yes we can," not negative
+  - Never condescending
```

Comment:
Curious to see if others agree: should we add in a paragraph about writing for all grade reading levels? This has been a best practice is my previous org, and given that HPE's customer base is global (and probably very busy!) it might be a good idea to include here. We could say something like:

"Consider writing for all reading levels to ensure clear, straightforward and efficient communication. A good benchmark to consider while writing is a United States grade 7 reading level. Check for grade reading level using an app like http://www.hemingwayapp.com/"